schools/make_demo.py

```diff
@@ -104,6 +107,56 @@ def make_student_group():
 		sg_tool.courses = sg_tool.get_courses()
 		sg_tool.create_student_groups()
 
+def make_examinations():
+	for i in xrange(1,12):
+		exam = frappe.new_doc("Examination")
+		exam.exam_name = "Examination" + str(i)
+		exam.exam_code = "EXAM00" + str(i)
+		temp_sg = random.choice(frappe.db.get_list("Student Group" , fields=("course" , "name")))
+		exam.course = temp_sg.course
+		exam.student_group = temp_sg.name
+		temp_inst = frappe.db.get_list("Instructor" , fields = ("instructor_name" , "name"))
+		t1 = random.choice(temp_inst)
+		t2 = random.choice(temp_inst)
+		exam.supervisor = t1.name
+		exam.supervisor_name = t1.instructor_name
+		exam.examiner = t2.name
+		exam.examiner_name = t2.instructor_name
+		exam.room = get_random("Room")
```

Comment:
You need to make sure the room is free and available (check against course schedule)